test/e2e/atlas/cleanup_test.go

```diff
@@ -38,19 +39,27 @@ func TestCleanup(t *testing.T) {
 		"-o=json")
 	cmd.Env = os.Environ()
 	resp, err := cmd.CombinedOutput()
-	req.NoError(err, resp)
+	req.NoError(err, string(resp))
 
 	var projects mongodbatlas.Projects
 	err = json.Unmarshal(resp, &projects)
 	req.NoError(err)
 
 	for _, project := range projects.Results {
-		if project.ID == os.Getenv("MCLI_PROJECT_ID") {
-			fmt.Println("skipping project", project.ID)
-			continue
-		}
```

Comment:
it now uses `t.Skip("skipping project", projectID)` instead of the break to make it more in line with the test framework, initially I was just gonna move the print to use t.Log but then I realized that t.Skip was maybe even better to flag the intention 
it even shows a
![Screenshot 2023-04-12 at 16 23 28](https://user-images.githubusercontent.com/461027/231505241-0ef7ed67-859f-4aab-83ae-9bbe785f5227.png)
s a nice skipped on the test report now

Reply:
thanks for clarifications, missed the t.skip below